test/e2e/reachability.go

```diff
@@ -230,6 +230,19 @@ func NewReachability(pods []Pod, defaultExpectation PodConnectivityMark) *Reacha
 	return r
 }
 
+func (r *Reachability) NewReachabilityWithSameExpectations() *Reachability {
+	var items []string
+	for _, pod := range r.Pods {
+		items = append(items, string(pod))
+	}
+	return &Reachability{
+		Expected:        r.Expected,
```

Comment:
I probably missed something, when does this `r.Expected` being initialized?

Reply:
When a new `Reachability` object is created, the default expected connectivity is set.